docs/en/datasets/classify/imagewoof.md

```diff
@@ -59,18 +59,22 @@ ImageWoof dataset comes in three different sizes to accommodate various research
 
 To use these variants in your training, simply replace 'imagewoof' in the dataset argument with 'imagewoof320' or 'imagewoof160'. For example:
 
-```python
-from ultralytics import YOLO
+!!! Example "Example"
 
-# Load a model
-model = YOLO("yolov8n-cls.pt")  # load a pretrained model (recommended for training)
+    === "Python"
```

Comment:
To keep consistency with the example above I would also add an example with the Ultralytics Command Line Interface here.

Reply:
Added CLI!